src/contracts/identity/context.tsx

```diff
@@ -96,28 +99,34 @@ const IdentityContractProvider = ({ children }: Props) => {
     }
 
     const getChainInfo = async (
-      rpcUrl: string
+      rpcUrls: string[]
     ): Promise<NetworkConsts | null> => {
+      const rpc = rpcUrls[0];
```

Comment:
Can this at least be random? It doesn't make a lot of sense to store multiple RPCs if we only use the first one.